test/hotspot/jtreg/vmTestbase/vm/mlvm/cp/share/GenFullCP.java

```diff
@@ -238,7 +238,7 @@ protected void createBootstrapMethod(ClassWriter cw, boolean isStatic, String me
     public Klass[] generateBytecodes() {
 
         // COMPUTE_FRAMES were disabled due to JDK-8079697
```

Comment:
Please remove this outdated comment as well. Also remember to update the copyright year.